docs/content/doc/usage/reverse-proxies.en-us.md

```diff
@@ -175,7 +175,7 @@ In case you already have a site, and you want Gitea to share the domain name, yo
 
 Then you **MUST** set something like `[server] ROOT_URL = http://git.example.com/git/` correctly in your configuration.
 
-Note: The following Apache HTTPD mods must be enabled: `proxy`, `proxy_http`.
+Note: The following Apache HTTPD mods must be enabled: `proxy`, `proxy_http`, additionally `lbmethod_byrequests` is required if proxying to https.
```

Comment:
Is `lbmethod_byrequests` really related? I can not see why it's required.

Reply:
You're right.  It is working without it.  I was following a guide somewhere else online and they said to enable it (and by extension load-balancer).